modules/representation_management/app/models/representation_management/form_2122_data.rb

```diff
@@ -20,7 +20,7 @@ def find_organization_name
     end
 
     def organization_name_resolves?
```

Comment:
Can you please make this `#organization_exists?` to match `representative_exists?`?

Reply:
Done!